File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.py
class Solution(object):
    def removeDuplicates(self, nums):
        for i in range(len(nums)):
            j=i+1
            while j<len(nums):
                if nums[i]==nums[j]:
                    nums.pop(j)
                else:
                    j+=1
        l=len(nums)
        return l
        """
        :type nums: List[int]
        :rtype: int
        """
        

File: 0026-remove-duplicates-from-sorted-array/0026-remove-duplicates-from-sorted-array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) 
    {
        int l=nums.size();
        

        for (int i=0 ; i<l; i++)
        {
            int j = i+1;
            while (j<l)
            {
                if ( nums[i]==nums[j])
                {
                    nums.erase(nums.begin()+(j-1));
                    l--;
                }
                else
                {
                    j++;
                }
                
            }

        }
        return nums.size();
        
    }
};